### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by joelondon**
further elaborating the file paths of some of the files modified to enable gla-WMS to gemini conversion

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Set up the harvest as above. Save and harvest the records.
 
 ### Add the directive to the batch processing config
 
-Edit the file `catalog\config\batch-process-cfg.json` and add the following entry:
+Edit the file `web-ui/src/main/resources/catalog/config/batch-process-cfg.json` and add the following entry:
```
Comment (archaeogeek) [inline README.md:34]: Only valid if you're building from source

File: README.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ Edit the file `catalog\config\batch-process-cfg.json` and add the following entr
 
 Note that the key must exactly match the name of the xsl.
 
-To provide a translated label for the process, edit `catalog\locale\en-custom.json` and add the following:
+To provide a translated label for the process, edit `web/target/geonetwork/catalog/locales/en-custom.json` and add the following:
```
Comment (archaeogeek) [inline README.md:44]: only valid if you're building from source

**Comment by archaeogeek** [comment]
@joelondon I'm in two minds about hard-coding the file paths in- maybe it would be better to put something at the top saying that if you're building from source it's in location x, if you're using tomcat it's location y and so on

**Comment by joelondon** [comment]
i got it working in tomcat's webapps/geonetwork dir

**Pull request closed without merging**